webapp/.eslintrc.json

```diff
@@ -150,13 +145,11 @@
         ]
       }
     ],
-    "indent": [
+    "import-newlines/enforce": [
       2,
-      4,
-      {
-        "SwitchCase": 0
-      }
+      3
     ],
+    "indent": 0,
```

Comment:
I think any rules that are disabled in this PR should be re-enabled, especially if the lints still pass without any further changes. Maybe we can have the linter avoid looking at the `mattermost-webapp` directory? Maybe we need to put it into `node_modules` to have it do that out of the box, or we can exclude it some other way